@here/olp-sdk-dataservice-read/lib/client/IndexLayerClient.ts

```diff
@@ -44,12 +44,24 @@ export interface IndexLayerClientParams {
  */
 export class IndexLayerClient {
     private readonly apiVersion: string = "v1";
-    // The HERE Resource Name of the catalog
-    private hrn: string;
-    // The ID of the layer.
-    private layerId: string;
-    // The [[OlpClientSettings]] instance.
-    private settings: OlpClientSettings;
+
+    /**
+     * HRN of the catalog.
+     * @deprecated This field will be marked as private by 05.2020.
```

Comment:
4-6 months. So it should be at least 08.2020.

Reply:
updated